README.md

```diff
@@ -25,25 +25,32 @@ Even if some checkbox is marked it does not mean that feature works perfectly.
 Some of those features are a likely outside the scope of this project, we are
 still learning and exploring what's possible.
 
-- [x] Linting with Scalafix on compile (textDocument/publishDiagnostics)
-- [ ] Linting with Scalafix as you type (textDocument/publishDiagnostics)
-- [ ] Refactoring with Scalafix (textDocument/codeAction)
-- [x] Formatting with Scalafmt (textDocument/formatting)
-- [ ] Formatting with Scalafmt for range (textDocument/rangeFormatting)
-- [ ] Formatting with Scalafmt as you type (textDocument/onTypeFormatting)
-- [x] Auto completions as you type with presentation compiler (textDocument/completions)
-- [x] Show type at position as you type (textDocument/hover)
-- [x] Go to definition from project Scala sources to project Scala sources on compile (textDocument/definition)
-- [x] Go to definition from project sources to Scala dependency source files on compile (textDocument/definition)
-- [x] Go to definition from project sources to Java dependency source file on compile (textDocument/definition)
-- [ ] Go to definition as you type (textDocument/definition)
-- [x] Show parameter list as you type (textDocument/signatureHelper)
-- [ ] Show red squigglies as you type (textDocument/publishDiagnostics)
-- [ ] Show red squigglies on compile (textDocument/publishDiagnostics)
-- [ ] Auto-insert missing import when completing a global symbol (textDocument/completions)
-- [ ] Find symbol references (textDocument/references)
-- [ ] Rename local symbol (textDocument/rename)
-- [ ] Rename global symbol (textDocument/rename)
+* Formatting with Scalafmt:
+  - [x] Whole file (`textDocument/formatting`)
+  - [ ] Selected range (`textDocument/rangeFormatting`)
+  - [ ] As you type (`textDocument/onTypeFormatting`)
+* Context-aware code completion:
```

Comment:
I agree "intellisense" is maybe too microsoft specific, but they use it as a general term for exactly these features 

> IntelliSense is the general term for a number of features: List Members, Parameter Info, Quick Info, and Complete Word. These features help you to learn more about the code you are using, keep track of the parameters you are typing, and add calls to properties and methods with only a few keystrokes.

Reply:
I see your point about multiple connotations, but I'm not sure whether it's a commonly understandable term. I had a general idea when I saw this word before, but when you mentioned it yesterday I had to google the exact meaning because I wasn't sure. Probably it's just me 😅